src/components/modals/save-form.component.tsx

```diff
@@ -82,15 +85,22 @@ const SaveForm: React.FC<SaveFormModalProps> = ({ form, schema }) => {
     }
   }, []);
 
-  const handleSubmit = async (event) => {
+  const handleSubmit = async (event: SyntheticEvent) => {
```

Comment:
Couldn't this be specified as `SyntheticEvent<{ name: { value: string };}>` etc?